migrate/v0_16/legacyaccounts/accounts.go

```diff
@@ -0,0 +1,485 @@
+package legacyaccounts
```

Comment:
This is used in the migrate test instead of the legacy types from the cosmos-sdk.  The cosmos-sdk types do not implement Spendable, etc methods.